modules/wildfly/pom.xml

```diff
@@ -61,11 +61,6 @@
             <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
```

Comment:
I dont know, none of our projects requires hibernate-infinispan and we never include it in modules or dist.